kitchen-sink-example.yaml

```diff
@@ -1,5 +1,40 @@
 # The file format version
 file_format: "0.1"
+exporters: &otlp-exporter
+  otlp:
+      # Sets the protocol.
+      #
+      # Environment variable: OTEL_EXPORTER_OTLP_PROTOCOL, OTEL_EXPORTER_OTLP_METRICS_PROTOCOL
+      protocol: http/protobuf
```

Comment:
I think the keys here are indented an extra two spaces?

Reply:
Fixed, thanks for catching